apps/consumer-client/src/pages/examples/add-pcd.tsx

```diff
@@ -637,9 +656,18 @@ async function addGPCPCD(
       argumentType: ArgumentTypeName.String,
       value: gpcConfig
     },
-    pod: {
-      value: await PODPCDPackage.serialize(podPCD),
-      argumentType: ArgumentTypeName.PCD
+    pods: {
+      value: {
+        podZero: {
```

Comment:
`examplePOD` maybe?  Not sure why `podZero` would be a meaningful name if it's not for counting 0, 1, 2.